src/runtime.js

```diff
@@ -240,9 +240,34 @@ function adoptShared(global, callback) {
         if (typeof fnOrStruct == 'function') {
           fnOrStruct(global.AMP);
         } else {
-          extensions.registerExtension(fnOrStruct.n, fnOrStruct.f, global.AMP);
+          // We support extension declarations which declare they have an
```

Comment:
iniPromise in line 239 should not delay preloading of deps

Reply:
done.